packages/ffe-cards-react/src/CardBase.spec.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import CardBase from './CardBase';
+
+const getWrapper = props => shallow(<CardBase {...props} />);
+const children = <div>Hello world</div>;
+
+describe('CardBase', () => {
+    it('should render a button element with correct class and children inside', () => {
```

Comment:
No longer renders a button - could you please update the text as well?

Reply:
Good catch! For sure, will do that when I squash commits :)
(I'll do the deprecation note for the old cards as well)